platform/email/src/main/resources/webapp/js/controllers.js

```diff
@@ -100,6 +100,29 @@
     controllers.controller('EmailSettingsCtrl', function ($scope, SettingsService) {
         $scope.settings = SettingsService.get();
 
+        $scope.add = function (property) {
+            if ($scope.settings.additionalProperties[property.name] === undefined) {
+                $scope.settings.additionalProperties[property.name] = property.value;
+                $scope.property = {};
+            } else {
+                motechAlert('email.header.error', 'email.settings.alreadyExist');
+            }
+            //console.log($scope.settings);
```

Comment:
Do not leave commented out lines of code. If you want to keep it and log to the console, add some sort of information what is being logged.